### PR TITLE
Add a filter featured to content articles

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
At the moment, it is not possible to filter for unfeatured articles in the content.
This PR adds a filter "featured" to the articles view

### Testing Instructions
You should have at least one fieatured and at least one unfeatured article. 
Apply the Patch and the list of filters in the Table Options has a new filter "featured".
Apply the filter to your articles.

### Documentation Changes Required
yes .. new filter

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 1]
```diff
@@ -90,6 +90,8 @@ COM_CONTENT_FIELD_VALUE_USE_ARTICLE_SETTINGS="Use Article Settings"
 COM_CONTENT_FIELD_VERSION_LABEL="Revision"
 COM_CONTENT_FIELD_XREFERENCE_DESC="An optional reference used to link to external data sources."
 COM_CONTENT_FIELD_XREFERENCE_LABEL="External Reference"
+COM_CONTENT_FILTER_FEATURED_NO ="Unfeatured Articles"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_content.ini]:
```suggestion
COM_CONTENT_FILTER_FEATURED_NO="Unfeatured Articles"
```

File: administrator/language/en-GB/en-GB.com_content.ini [hunk 2]
```diff
@@ -90,6 +90,8 @@ COM_CONTENT_FIELD_VALUE_USE_ARTICLE_SETTINGS="Use Article Settings"
...
+COM_CONTENT_FILTER_FEATURED_YES ="Featured Articles"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.com_content.ini]:
```suggestion
COM_CONTENT_FILTER_FEATURED_YES="Featured Articles"
```

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 5f1f8b3adaafa3a4f0a9fe84bff0e27c1d1c7e35<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23497">issues.joomla.org/tracker/joomla-cms/23497</a>.</sub>

**Comment by ghazal** [comment]
I have tested this item :white_check_mark: successfully on 5f1f8b3adaafa3a4f0a9fe84bff0e27c1d1c7e35<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23497">issues.joomla.org/tracker/joomla-cms/23497</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23497">issues.joomla.org/tracker/joomla-cms/23497</a>.</sub>

**Comment by HLeithner** [comment]
@chmst could you please upgrade this branch

**Comment by wilsonge** [comment]
Thanks :)

**Pull request merged**